running-coreos/bare-metal/installing-to-disk/index.md

```diff
@@ -15,14 +15,27 @@ weight: 7
 There is a simple installer that will destroy everything on the given target disk and install CoreOS.
 Essentially it downloads an image, verifies it with gpg and then copies it bit for bit to disk.
 
-The script is self-contained and located [on GitHub here](https://raw.github.com/coreos/init/master/bin/coreos-install "coreos-install") and can be run from any Linux distribution. You cannot install CoreOS to the same device that is currently booted. However, the [CoreOS ISO]({{site.url}}/docs/running-coreos/platforms/iso/) or any Linux liveCD will allow CoreOS to install to a non-active device. 
+The script is self-contained and located [on GitHub here](https://raw.github.com/coreos/init/master/bin/coreos-install "coreos-install") and can be run from any Linux distribution. You cannot normally install CoreOS to the same device that is currently booted. However, the [CoreOS ISO]({{site.url}}/docs/running-coreos/platforms/iso/) or any Linux liveCD will allow CoreOS to install to a non-active device. 
 
-If you already boot CoreOS via PXE, the install script is already installed. By default the install script will attempt to install the same version and channel that was PXE-booted:
+If you boot CoreOS via PXE, the install script is already installed. By default the install script will attempt to install the same version and channel that was PXE-booted:
 
 ```sh
 coreos-install -d /dev/sda
 ```
 
+If you are using the ISO with VMware, first sudo to root: 
+
+```sh
+sudo su - root
+```
+
+Then install as you would with the PXE booted system, but be sure to include user information, especially an ssh key, in a Cloud-Config file, or else you will not be able to log into your CoreOS instance.  
```

Comment:
Rob -

Thanks for your comments. I've made the changes and pushed them to my repo.
I'm new to the process, shall I make an updated pull request or is that
sufficient?

Dave

On Tue, Apr 28, 2015 at 10:34 PM, Rob Szumski <notifications@github.com>
wrote:

> In running-coreos/bare-metal/installing-to-disk/index.md
> <https://github.com/coreos/docs/pull/492#discussion_r29311356>:
>
> >
> >  ```sh
> >  coreos-install -d /dev/sda
> >  ```
> >
> > +If you are using the ISO with VMware, first sudo to root:
> > +
> > +```sh
> > +sudo su - root
> > +```
> > +
> > +Then install as you would with the PXE booted system, but be sure to include user information, especially an ssh key, in a Cloud-Config file, or else you will not be able to log into your CoreOS instance.
>
> Can you link to the cloud-config header below:
> [cloud-config](#cloud-config)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/docs/pull/492/files#r29311356>.
>